bundles/net.tourbook/src/net/tourbook/ui/views/nutrition/DialogSearchProduct.java

```diff
@@ -0,0 +1,853 @@
+/*******************************************************************************
+ * Copyright (C) 2024 Frédéric Bard
+ *
+ * This program is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU General Public License as published by the Free Software
+ * Foundation version 2 of the License.
+ *
+ * This program is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License along with
+ * this program; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110, USA
+ *******************************************************************************/
+package net.tourbook.ui.views.nutrition;
+
```

Comment:
I think it looks better in this case, that the first letter of `name` and `barcode` should be written in uppercase

By Name
By Barcode

![image](https://github.com/mytourbook/mytourbook/assets/1283445/d9c198a7-85ad-4a9b-882c-57e2c615d3f5)

Reply:
Fixed